src/main/java/seedu/address/model/wedding/Wedding.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.model.wedding;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.model.person.PersonId;
+
+/**
+ * Represents a wedding event and tracks the date and people related to the event, immutable
+ */
+public class Wedding {
+    private final String name;
+    private final String date;
+    private final List<PersonId> assignees; //Stores a list of UserIDs
```

Comment:
Should this be a list of Persons instead? Since Wedding is a container for Person objects, rather than PersonId objects.